src/Components/DisplayOfArticles/ArticleCard/index.jsx

```diff
@@ -0,0 +1,39 @@
+import "./ArticleCard.css";
+import { getAllArticles } from "../../../utils/api";
+import { useState } from "react";
+import { useEffect } from "react";
+
+export function ArticleCard() {
+  const [newArticles, setNewArticles] = useState([]);
+
+  useEffect(() => {
+    getAllArticles().then((data) => {
+      console.log(data);
+      setNewArticles(data.articles);
+    });
+  }, []);
+
+  return (
+    <ul className="card-list">
+      {" "}
+      {newArticles.map((article) => {
+        return (
+          <li key={article.article_id}>
```

Comment:
The return of this map could be extracted out into an _ArticleCard_ component to arrange and display each element in the list